syft/execution/plan.py

```diff
@@ -490,8 +491,9 @@ def call_wrapped_framework(role):
 
     def add_translation(self, plan_translator: "AbstractPlanTranslator"):
         role = plan_translator(self).translate()
-        self.roles[plan_translator.framework] = role
-        return role
+        if isinstance(role, Role):
```

Comment:
Q: When is the case when ```role``` is not a ```Role```?

Reply:
Torchscript translator returns Plan: 
https://github.com/OpenMined/PySyft/blob/vvm/plan-roles-serialize/syft/execution/translation/torchscript.py#L50
It's different kind of translator that doesn't produce new "role" but sets torchscript property in Plan.
Well, I guess we'll eventually need to refactor to move that property to Role level.